### PR TITLE
[google_maps_flutter] Disable failing iOS tests

### PR DESCRIPTION
**Pull request by stuartmorgan**
Disables the tests that have started failing for unknown OOB reasons, in order to fix the tree.

https://github.com/flutter/flutter/issues/139825

### CODE REVIEW

**Review by camsim99 — APPROVED** [review]

File: packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart [hunk 1]
```diff
@@ -135,7 +137,9 @@ void runTests() {
 
     expect(firstVisibleRegion, isNot(secondVisibleRegion));
     expect(secondVisibleRegion.contains(newCenter), isTrue);
-  });
+  },
+      // TODO(stuartmorgan): Re-enable; see https://github.com/flutter/flutter/issues/139825
+      skip: true);
```
Comment (ditman) [inline packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart:142]:
```suggestion
      skip: isIOS);
```

File: packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart [hunk 2]
```diff
@@ -269,7 +273,9 @@ void runTests() {
     await tester.pumpAndSettle();
     zoom = await controller.getZoomLevel();
     expect(zoom, equals(7));
-  });
+  },
+      // TODO(stuartmorgan): Re-enable; see https://github.com/flutter/flutter/issues/139825
+      skip: true);
```
Comment (ditman) [inline packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart:278]:
```suggestion
      skip: isIOS);
```

File: packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart [hunk 3]
```diff
@@ -302,7 +308,9 @@ void runTests() {
     final ScreenCoordinate topLeft =
         await controller.getScreenCoordinate(northWest);
     expect(topLeft, const ScreenCoordinate(x: 0, y: 0));
-  });
+  },
+      // TODO(stuartmorgan): Re-enable; see https://github.com/flutter/flutter/issues/139825
+      skip: true);
```
Comment (ditman) [inline packages/google_maps_flutter/google_maps_flutter/example/integration_test/src/maps_controller.dart:313]:
```suggestion
      skip: isIOS);
```

**Review by ditman — APPROVED** [review]
LGTM! but maybe only skip if `isIOS` on the core plugin?

**Comment by stuartmorgan** [comment]
Doh, I meant to make those iOS when switching plugins and then completely forgot. Thanks!

**Pull request merged**